template/references/elements/todo.md.twig

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
This file is never used nor imported. It seems a copy/paste forgotten, is it still useful? Otherwise, don't forget to remove it